packages/ckeditor5-dev-tests/bin/notify-travis-status.js

```diff
@@ -176,51 +174,77 @@ function getFormattedMessage( message, repoOwner, repoName ) {
 		} );
 }
 
-/**
- * Returns a URL to GitHub API which returns details of the commit that caused the CI to fail its job.
- *
- * @returns {String}
- */
-function getGithubApiUrl() {
-	let commitUrl;
-
+function getCommitUrl() {
 	const { SLACK_NOTIFY_COMMIT_URL, TRAVIS_REPO_SLUG, TRAVIS_COMMIT } = process.env;
 
 	if ( SLACK_NOTIFY_COMMIT_URL ) {
-		commitUrl = SLACK_NOTIFY_COMMIT_URL;
+		return SLACK_NOTIFY_COMMIT_URL;
 	} else {
-		commitUrl = `https://github.com/${ TRAVIS_REPO_SLUG }/commit/${ TRAVIS_COMMIT }`;
+		return `https://github.com/${ TRAVIS_REPO_SLUG }/commit/${ TRAVIS_COMMIT }`;
 	}
```

Comment:
The `else` branch is not needed since the `if` ends with `return` statement.

My personal style in such cases is:

```js
return SLACK_NOTIFY_COMMIT_URL || `https://github.com/${ TRAVIS_REPO_SLUG }/commit/${ TRAVIS_COMMIT }`;
```